packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [3.4.0]
+* Updates to the way the project is being generated: now the project is generated using the `flutter create` command, building on top of the default generated files
```

Comment:
Can we change the change-log to something shorter. For example: "Use flutter create to generate the native iOS and Android folders to ensure compatibility in the future"